app/tests/test_assets_endpoint.py

```diff
@@ -554,25 +555,31 @@ def test_asset_endpoint_put_rename_asset(self):
         response = self.client.put(
             path, data=changed_asset.get_json('put'), content_type="application/json"
         )
-        self.assertStatusCode(200, response)
-        json_data = response.json()
-        self.assertEqual(changed_asset.json['id'], json_data['id'])
-        self.check_stac_asset(
-            changed_asset.json, json_data, collection_name, item_name, ignore=['item']
-        )
+        self.assertStatusCode(400, response)
+
+        self.assertStatusCode(400, response)
```

Comment:
Duplicates lines. It would good also to check after the status code that the description is the expected one (e.g. `Renamed not allowed`) to make sure that we got a 400 for the good reason (there is so many reasons for 400).